students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_2/to_swap_the_two_words.py

```diff
@@ -0,0 +1,5 @@
+two_word_message = "Run Forrest"
+space_position = two_word_message.find(" ")
```

Comment:
Why not this? https://docs.python.org/3/library/stdtypes.html#str.split

Reply:
It was specified in the task to use "count" method.
"Given a string consisting of words separated by spaces. Determine how many words it has. To solve the problem, use the method count."